src/templates/pycontw-2020/contents/en/events/overview.html

```diff
@@ -45,9 +46,35 @@ <h2>Open Spaces</h2>
 
 <p>More information about <em>Open Spaces</em> can be found on the <a href="{{ events_openspaces_url }}">{% trans 'Open Spaces' %}</a> page.</p>
 
+
 <h2>Job Fair</h2>
 
 <p>Job Fair is a career expo for Python developers. During the Job Fair session, our sponsors will take turns introduce themselves on the stage and promote their job openings. For attendees that are looking for a career, this is a great opportunity for you to interact with the sponsors or submit your rèsumè!</p>
 
+
+<h2>Community Track</h2>
+
+<p>The community track is a new event in PyCon TW. We choose the talks with specific topics and package them into the same session. We'll leave the conference rooms and go to characteristic cafes, exhibition spaces, and even monuments so that the attendees and speakers can interact with each other more comfortably and closely.</p>
+
+<p>In short, the "community track" is to build strong connections between attendees, speakers, sponsors, and communities!</p>
+
+<p>The community track will take place in the afternoon of day 2 (9/6), with a duration of 3 hours, including 90-minute talks arranged by PyCon Taiwan.</p>
+
+<p>More information about <em>Community Track</em> can be found on the <a href="{{ events_talk_list_url }}">{% trans 'Talks' %}</a> page.</p>
+
+
+<h2>Sprints</h2>
+
+<p>Sprints are held out of the main conference. A sprint event gathers open source project owners, contributors, and people who wants to contribute but trying to find a place to start. During a sprint, project hosts bring their unresolved issues or new features under development and share with everyone. You can join a project of your choice, or bring your own project here!</p>
+
+<p>For those who already knows fundamentals of Python and wants to participate open source projects, but doesn't know where to start, sprints are great opportunities to get yourself involved! As for project hosts, this is a great chance for you to find people with similar interests, having face-to-face interactions with contributors, and getting new contributors to your project's development! Be prepared to get your hands dirty!</p>
+
+<p>More information about <em>Sprint</em> can be found on the <a href="{{ events_sprint_url }}">{% trans 'Sprints' %}</a> page.</p>
```

Comment:
I think we could add registration information here.
https://pycontw.kktix.cc/events/20200830-sprint
https://hackmd.io/w5hh0hUNQr65k4ayG2128Q?view

Reply:
Done~